### DISCUSSION TITLE
⛔ Features that should exist but do not exist!

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21931] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

The new features are great and amazing. Especially AI😉 But some important features are still missing in Elementor. Please do not forget them.

### Describe the solution you'd like

Features that exist in other page builders, but Elementor does not have them. As :

1. Responsive mode for custom css
2. Gradient color for texts, especially the heading widget
3. Responsive mode for colors
4. Relative and sticky positions

The above are very important in their own way and new page builders benefit from all of the above. Please keep them in mind and introduce them along with new and wonderful features. **If friends have other features in mind and I forgot to write them, add them.**

thanks

**Comment by ayoubkhan558** [comment]
Upvote for +1
Responsive mode for custom CSS

**Comment by nicholaszein** [comment]
Hi @Joliamendi!

thanks for submitting your feature request! 🙏 

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

Kind regards
